plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -504,12 +504,23 @@ function OPDSBrowser:genItemTableFromCatalog(catalog, item_url, username, passwo
                             title = link.title,
                         })
                     elseif link.rel == self.stream_rel then
+                        -- This for loop iterates through all keys in a
+                        --   Entry and looks for the count tag, then stores
+                        --   That key to use for updating the table value.
+                        local count_key = ""
+                        for k, v in pairs(link) do
+                            if string.sub(k, -6) == ":count" then
+                                count_key = k
+                                break
+                            end
+                        end
+                        logger.dbg("---> count key is",count_key)
```

Comment:
Assuming we want to keep this debug line, please make it something closer to:
```suggestion
                        logger.dbg("OPDS catalog ---> count key is", count_key)
```